contracts/src/types/Requirements.sol

```diff
@@ -12,12 +12,19 @@ struct Requirements {
     bytes32 imageId;
     Callback callback;
     Predicate predicate;
+    /// @notice The selector for the seal.
+    /// @dev If the selector is set to zero, the seal is not required.
     bytes4 selector;
+    /// @notice The address of the IRiscZeroVerifier contract.
+    /// @dev If the address is set to zero, the default IRiscZeroVerifier is used.
+    address verifier;
```

Comment:
This is added if we want to give to the requestor full freedom to decide how its application seal should be verified.
Otherwise deploying via multi-router could also work